packages/core-data/src/test/selectors.js

```diff
@@ -160,14 +160,14 @@ describe( 'hasEntityRecords', () => {
 		expect( hasEntityRecords( state, 'root', 'postType' ) ).toBe( false );
 	} );
 
-	it( 'returns true if the entity configuration is not known', () => {
+	it( 'returns false if the entity configuration is not known', () => {
```

Comment:
This doesn't make sense to me either. Why would we want to return `true` for `hasEntityRecords` when the entity is still unknown?

Reply:
Yeah, agree, that doesn't seem right given the documentation for `hasEntityRecords`:
> Returns true if records have been received for the given set of parameters, or false otherwise.

The key word being 'received'. That queried data prepopulates for known entities is irrelevant, those records weren't received in the typical way we use that term.

I also find the name `hasEntityRecords` confusing since it should really be called `hasReceivedEntityRecords`. The current name would lead me to believe an empty array would return `false` too, but the function definition instead just checks `Array.isArray`.